### PR TITLE
Add button in now playing menu for lyrics.

### PR DESCRIPTION
**Pull request by jostrander**
Also modified the lyrics screen to fade in at 150ms, seemed a little less harsh after clicking the menu button.

### CODE REVIEW

File: src/inject/GPMInject/interface/customUI.js [hunk 1]
```diff
@@ -99,6 +99,51 @@ function installDesktopSettingsButton() {
   document.querySelectorAll('.nav-section.material')[0].insertBefore(dSettings, document.querySelectorAll('.nav-section.material > a')[2]); // eslint-disable-line
 }
 
+/* eslint-disable max-len, no-multi-str */
+function installNowPlayingButton(text, id, callback) {
+  const ripple = '<paper-ripple style="-webkit-user-select: none;"> \
```
Comment (MarshallOfSound) [inline src/inject/GPMInject/interface/customUI.js]: If you use backticks to surround the string you can go multi line without "\"

File: src/inject/GPMInject/interface/customUI.js [hunk 2]
```diff
@@ -99,6 +99,51 @@ function installDesktopSettingsButton() {
...
+                  <div id="background" class="style-scope paper-ripple" style="-webkit-user-select: none;"></div> \
+                  <div id="waves" class="style-scope paper-ripple" style="-webkit-user-select: none;"></div> \
+                  </paper-ripple>';
+  const content = document.createElement('div');
+  content.setAttribute('class', 'goog-menuitem-content');
+  content.innerHTML = `${ripple} ${text}`;
```
Comment (MarshallOfSound) [inline src/inject/GPMInject/interface/customUI.js]: With lot's of stuff you don't need to add in the ripple HTML, GPM somehow makes it for you.  Examples are the buttons we inject.  Did you try without adding the ripple?

Comment (jostrander) [inline src/inject/GPMInject/interface/customUI.js]: Just removed the ripple html, and no ripple :-1:

Comment (MarshallOfSound) [inline src/inject/GPMInject/interface/customUI.js]: :cry:

**Pull request merged**